src/porepy/models/solution_strategy.py

```diff
@@ -622,4 +635,4 @@ def update_time_dependent_ad_arrays(self, initial: bool) -> None:
                 updated by copying the iterate.
 
         """
-        pass
+        self.update_boundary_conditions(initial=initial)
```

Comment:
Again, the "initial" strongly suggests that "set" is better than "update"

Reply:
Agreed, changing it.